yamlfix.toml

```diff
@@ -0,0 +1,11 @@
+explicit_start = true
+line_length = 120
```

Comment:
It looks like it is being wrapped at 80 characters not 120.  80 is preferable as it match Google style for c++.  Does this do anything?

Reply:
Yes, it does. The command to run for check is `yamlfix -c yamlfix.toml flow/scripts/variables.yaml --check`, whereas for format is `yamlfix -c yamlfix.toml flow/scripts/variables.yaml`. Next PR should reflect line length 120.